packages/product-download-page/index.tsx

```diff
@@ -114,8 +115,9 @@ export default function ProductDownloadsPage({
       <HashiHead title={`Downloads | ${name} by HashiCorp`} />
       <div className={`${styles.root} ${themeClass || ''} ${className || ''}`}>
         <h1 className={styles.pageTitle}>
-          Download {name}
-          {enterpriseMode ? ' Enterprise' : ''}
+          {pageTitle ||
+            `Download ${name}
+          ${enterpriseMode ? ' Enterprise' : ''}`}
```

Comment:
Allows the useProductMeta-based page title to be overridden. See [the vagrant vmware utility download page](https://vagrant-git-fork-zchsh-zsupgrade-global-styles-hashicorp.vercel.app/vmware/downloads) for an example use case.